src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -291,14 +291,29 @@ export class BrowsingContextProcessor {
     return context.getOrCreateSandbox(target.sandbox);
   }
 
-  // eslint-disable-next-line @typescript-eslint/require-await
   async process_script_addPreloadScript(
-    _params: Script.AddPreloadScriptParameters
+    params: Script.AddPreloadScriptParameters
   ): Promise<Script.AddPreloadScriptResult> {
-    throw new Message.UnknownErrorException('Not implemented.');
+    const cdpClient = this.#cdpConnection.browserClient();
+
+    const result1 = await cdpClient.sendCommand('Runtime.callFunctionOn', {
```

Comment:
This line is not needed. On the browser client you don't have a runtime.

Reply:
Ack, removed.